frontend/src/components/CanteenPublication.vue

```diff
@@ -105,6 +121,24 @@ export default {
     earnedBadges() {
       return earnedBadges(this.canteen, this.diagnostic, this.$store.state.sectors)
     },
+    shouldDisplayGraph() {
```

Comment:
@hfroot I added this condition to make sure that 
a) we have diagnostics with appro data
b) we either have more than one (for historical display) or the one we have is different from last year